step-release-vis/src/app/services/data.ts

```diff
@@ -0,0 +1,43 @@
+import {Injectable} from '@angular/core';
```

Comment:
tests for this?

Reply:
ahh nvm the file rename confused me a bit.. checking there

Reply:
I added a _Service_ postfix to all services to avoid further confusion.